validator/src/main.rs

```diff
@@ -507,10 +507,10 @@ pub fn main() {
                 .help("Enable the JSON RPC 'setLogFilter' API.  Only enable in a debug environment"),
         )
         .arg(
-            Arg::with_name("enable_rpc_get_confirmed_block")
-                .long("enable-rpc-get-confirmed-block")
+            Arg::with_name("enable_rpc_transaction_info")
```

Comment:
I don't think this name covers the rewards tracking part of `get_confirmed_block`. I also feel like the name should be similar to `limit-ledger-size` since it's a bit related. Maybe `enable-ledger-history` or something?